README.md

```diff
@@ -1,3 +1,6 @@
 # Third-Party Libraries
 
 This repository contains certain open source third-party software used by the Unreal Engine.
+Draze BRN
```

Comment:
Remove names, this isn't useful for organizations and open source. Please be aware to follow the culture.